lib/wallaby/query.ex

```diff
@@ -29,6 +29,7 @@ defmodule Wallaby.Query do
     * `:count` - The number of elements that should be found (default: 1).
     * `:visible` - Determines if the query should return only visible elements (default: true).
     * `:text` - Text that should be found inside the element (default: nil).
+    * `:get` - The number of the element to return if multimple elements satisfy the selection criteria.
```

Comment:
Sorry - I should perhaps not have been flippant about laziness - but brevity IS a virtue. This is not code that will be written once, but a parameter name that will be written (hopefully) thousands of times. All else equal, pick the short one!  A shorter name just may help avoid tests spilling onto another line, and so improve readability.

OK - last offer - how about `item`? It reads well (item 5), carries the air of singularity that most of these suggestions lack AND ... is short. ;-)

Looking forward to keathley's view.